Riot/Model/RoomList/RecentsDataSource.m

```diff
@@ -634,6 +648,15 @@ - (void)refreshRoomsSections
         if (conversationCellDataArray.count > 0)
         {
             conversationSection = sectionsCount++;
+
+            if (_recentsDataSourceMode == RecentsDataSourceModeRooms)
+            {
+                // Add the directory section after "ROOMS"
+                directorySection = sectionsCount++;
+
+                // Make _publicRoomsDirectoryDataSource start loading data
+                _publicRoomsDirectoryDataSource.searchPattern = nil;
```

Comment:
Can we start loading publics into - (void)setRecentsDataSourceMode:(RecentsDataSourceMode)recentsDataSourceMode?